custom/enikshay/case_utils.py

```diff
@@ -55,6 +55,13 @@ def get_person_case_from_occurrence(domain, occurrence_case_id):
     return get_parent_of_case(domain, occurrence_case_id, CASE_TYPE_PERSON)
 
 
+def get_person_case_from_episode(domain, episode_case_id):
+    get_person_case_from_occurrence(
+        domain,
+        get_occurrence_case_from_episode(domain, episode_case_id).get_id
```

Comment:
I think you want to do `case_id` instead of `get_id` here. I don't think that `CommCareCaseSQL` works with `get_id`.